modules/tests/src/test/scala/derevo/tests/HigherKindedSuite.scala

```diff
@@ -19,7 +19,8 @@ trait Zee[A, F[_]] {
 @derive(Cody)
 case class Nee[F[_]](greeting: F[String])
 
-@derive(Cody)
+// Test for qualified annotation name
+@derevo.derive(Cody)
```

Comment:
Does it cover `_root_.derevo.derive`?
What about
```
import derevo.{derive => lolrive}

@lolrive(Cody)
```
```

Reply:
I think there's a better solution by not matching `derive` at all (`new $_(..$instances)` should work)